src/test/unit/sanitize.test.js

```diff
@@ -571,21 +564,3 @@ describe('sanitizePII', function () {
     expect(sanitizedProfile.threads[0]['eTLD+1']).toBe(eTLDPlus1);
   });
 });
-
-describe('getRemoveProfileInformation', function () {
```

Comment:
I decided to move this to the other file, as this was working with stores and the publish selectors. I did that in its separate commit, but I didn't touch this test at all otherwise.